package.json

```diff
@@ -6,7 +6,7 @@
   "homepage": "/min-side-arbeidsgiver",
   "dependencies": {
     "@craco/craco": "^5.6.4",
-    "@navikt/bedriftsmeny": "^3.0.2",
+    "@navikt/bedriftsmeny": "^3.0.3",
```

Comment:
burde vi kanskje ikke ha semver range på disse?

Reply:
kommentar bare for diskusjon

Reply:
Hva anbefaler du?

Reply:
sette nøyaktig versjon uten semver range `"3.0.3"`